src/System.Security.Cryptography.Encoding/tests/Asn1/Reader/ReadNamedBitList.cs

```diff
@@ -163,9 +163,9 @@ public static void ReadNamedBitList_RequiresFlags(PublicEncodingRules ruleSet)
             string inputHex = "030100";
             AsnReader reader = new AsnReader(inputHex.HexToByteArray(), (AsnEncodingRules)ruleSet);
 
-            AssertExtensions.Throws<ArgumentException>(
+            Assert.Throws<ArgumentException>(
```

Comment:
All of these `AssertExtensions` => `Assert` changes should be undone; it's needed for the AoT configurations.

Reply:
Are you sure about that? It is already used in other unit tests in the S.S.C.* assemblies. I was pretty sure it's not necessary after Xunit was updated couple of months ago.

Reply:
My understand was the big sweeps like https://github.com/dotnet/corefx/commit/59915687bb261efeae70565d0ca8990ac45ceb9c#diff-cc0921e154e60f6f28606ee1a4685a39 were for .NET Native.

https://source.dot.net/#CoreFx.Private.TestUtilities/System/AssertExtensions.cs,81 suggests that, too.

Reply:
These tests are hitting implementation that is residing in the same assembly, so presumably it should work just fine on .NET Native (and CI is green). I do see reason to use `AssertExtensions` for cases where system API is tested and the actual parameter names between CoreFX/NetFX/ProjectN differ for historical or other reason.

I cannot track down the original discussion that lead to the decision to use `AssertExtensions` accross the board. @danmosemsft, @hughbe some pointers? The issue #21409 was closed more than year ago claiming that the work has completed. Some new CoreFX-only code uses the `Assert.Throws<ArgumentException>(paramName, ...)` overloads. If the guidance is to use `AssertExtensions` across the board then I will change it back, but the linked issue should be reopened to track the new usages that creeped back in.

Reply:
Whether or not it *needs* to change back; please remove any unnecessary edits... this being one of them.